patches/net/minecraft/client/renderer/RenderBuffers.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/client/renderer/RenderBuffers.java
++++ b/net/minecraft/client/renderer/RenderBuffers.java
+@@ -39,6 +_,7 @@
+             put(p_307112_, RenderType.waterMask());
+             ModelBakery.DESTROY_TYPES.forEach(p_173062_ -> put(p_307112_, p_173062_));
+         });
++        net.neoforged.fml.ModLoader.get().postEvent(new net.neoforged.neoforge.client.event.RegisterRenderBuffersEvent(sortedmap));
```

Comment:
We generally fire client-side registration events in `ClientHooks.init`. Not sure about the location, maybe it's fine as it is in the PR.

Reply:
The problem is that I need to supply the map of fixed buffers that is populated only inside RenderBuffers constructor to the event. So I'm not sure if it's possible to fire it anywhere else

Reply:
I would say firing it from the vanilla code is fine in this case as it happens from a constructor and not from "random" static init and said constructor is guaranteed to fire after initial mod loading is already done.

Reply:
Fine by me. :)